Analysis/Occurance.cs

```diff
@@ -0,0 +1,9 @@
+namespace BSDetector {
+    public class Occurance {
```

Comment:
Shouldn't it be named "Occurrence"?

Reply:
You missed one "r". 😄

Reply:
Commits at 11pm rarely solve a thing :satisfied: